doc/source/dev/contributor/building.rst

```diff
@@ -0,0 +1,182 @@
+
+        To install SciPy build requirements, you can do::
+
+          sudo yum install gcc-gfortran python3-devel openblas-devel lapack-devel
+
+        Alternatively, you can do::
+
+          sudo yum-builddep scipy
+
+        This command installs whatever is needed to build SciPy, with the
+        advantage that new dependencies or updates to required versions are
+        handled by the package managers.
+
+      .. tab:: Arch
+
+        To install SciPy build requirements, you can do::
+
+          sudo pacman -S gcc-gfortran python-devel openblas-devel lapack-devel
+
+    All further work should proceed in a virtual environment. Popular options
+    include the standard library ``venv`` module or a separate ``virtualenv``
+    package. 
+
+    Typically, you will want to install all of the above from packages supplied by
+    your Linux distribution, as building them yourself is complicated. If you need
+    to use specific BLAS/LAPACK libraries, you can do
+
+    ::
+
+        export BLAS=/path/to/libblas.so
+        export LAPACK=/path/to/liblapack.so
+        export ATLAS=/path/to/libatlas.so
+
+    * The `Cython <https://cython.org/>`__ and
+      `Pythran <https://pythran.readthedocs.io>`__ ahead-of-time compilers are also
+      necessary, as is ``pybind11``. It is recommended to install these packages
+      with ``pip`` or ``conda``, because it is possible (even likely) that you need
+      newer versions of these packages than the ones that are available in your
+      Linux distribution.
+
+    If you are using conda, these dependencies can be installed in the conda
+    environment itself. See :ref:`conda-guide` for more details.
+
+  .. tab:: OSX
+
+    Install Apple Developer Tools. An easy way to do this is to
+    `open a terminal window <https://blog.teamtreehouse.com/introduction-to-the-mac-os-x-command-line>`_,
+    enter the command
+
+    ::
+
+        xcode-select --install
+
+    and follow the prompts. Apple Developer Tools includes
+    `git <https://git-scm.com/>`_, the software we need to download and manage the
+    SciPy source code.
+
+    We recommend :ref:`using conda <conda-guide>` to set up your environment on a
+    Mac, as this will also allow you to install all necessary dependencies in a
+    conda environment, instead of manually. If you prefer not using conda, see
+    :ref:`build-osx`.
+
+  .. tab:: Windows
+
+    We recommend :ref:`using conda <conda-guide>` to set up your environment on
+    Windows, as this will also allow you to install all necessary dependencies
+    in a conda environment, instead of manually. If you prefer not using conda
+    or need more details, see :ref:`build-windows`.
+
+Detailed instructions
+---------------------
+
+.. toctree::
+   :maxdepth: 1
+
+   osx
+   windows
+   meson
+   meson_advanced
+   building_faq
+
+
+Reference for build options
+===========================
+
+SciPy has several tunable build-time options, which can be set.
+
```

Comment:
This all works differently with Meson, but let's do that in a second iteration. How about for now:
```
.. warning::

    This content is for the old `numpy.distutils`-based build and doesn't apply to the
    Meson build (i.e., when building with `python dev.py`).
```